modules/tinymce/src/themes/silver/main/ts/ui/general/OuterContainer.ts

```diff
@@ -115,6 +119,22 @@ const factory: UiSketcher.CompositeSketchFactory<OuterContainerSketchDetail, Out
       const toolbar = Composite.parts.getPart(comp, detail, 'toolbar');
       toolbar.each((toolbar) => toolbar.getApis<ToolbarApis>().refresh(toolbar));
     },
+    toggleToolbar(comp) {
+      // toggle may not be defined on all toolbars e.g. 'scrolling' and 'wrap'
+      Composite.parts.getPart(comp, detail, 'toolbar')
+        .each((toolbar) => {
```

Comment:
Could we do something like this instead?

```ts
Composite.parts.getPart(comp, detail, 'toolbar')
  .bind((toolbar) => Optional.from(toolbar.getApis<ToolbarApis>().toggle))
  .each((toggle) => toggle(toolbar));
```

I read `.each` as "we're explicitly doing a side-effect here" which isn't really the case for the first `each` call

Reply:
Each is to me is "this doesn't return anything" more than "this has side-effects", in which case this makes sense as is. It does feel a little odd to do `each((toolbar) => Optional.from...` though...